webrtc.html

```diff
@@ -5236,6 +5240,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <var>sender</var>.</p>
                 </li>
                 <li>
+                  <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                  <code>true</code>, abort these steps.</p>
+                </li>
+                <li>
```

Comment:
Great! I missed this one. I was actually thinking of step 10.3 below here:

*"Queue a task that sets sender's track attribute to withTrack, and resolves p with undefined."*